examples/nodejs/sites/create.js

```diff
@@ -4,7 +4,7 @@ const { Site } = require('calibre')
 
 const create = async () => {
   const name = 'Calibre'
-  const location = 'Frankfurt'
+  const location = 'Local'
```

Comment:
This might've slipped through from dev mode

Reply:
That was quick! Will update.